src/main/java/net/openhft/chronicle/threads/Threads.java

```diff
@@ -41,7 +41,7 @@ public enum Threads {
     ; // none
 
     private static final int MAX_DEPTH_TO_FOLLOW_DELEGATIONS = 20;
-    static final Field GROUP = Jvm.getField(Thread.class, "group");
+    static Field GROUP;
```

Comment:
Actually sonar made a good point here, we should probably rename it to match the fact it's no longer final. But perhaps we should also make it private because anyone accessing it might be surprised to find that it's now nullable.

Reply:
Agree. Have made it private which is a breaking change but balanced by the fact that it is very unlikely that anyone is using